apps/main/src/global-extensions.ts

```diff
@@ -0,0 +1,13 @@
+import type { UTCTimestamp } from 'lightweight-charts'
+
+declare global {
+  interface Date {
+    getUTCTimestamp(): UTCTimestamp
```

Comment:
This does nothing related to UTC.
For consistency with `getTime` I would suggest
```suggestion
    getTimeInSeconds(): UTCTimestamp
```

Reply:
It also converts from Javascript's milliseconds to 'normal' UTC timestamp in seconds, as expected by charting libraries. Given the prices API now uses Date's that's a lot of instances of `(x.getTime() / 1000) as UTCTimestamp` without the prototype. The casting is necessary btw, as it's a nominal type and lightweight charts doesn't accept a raw `number`.